frontend/packages/kubevirt-plugin/src/constants/status-enum.ts

```diff
@@ -95,20 +95,23 @@ export abstract class StatusEnum<SIMPLE_LABEL = StatusSimpleLabel> extends Objec
 
   getSimpleLabel = () => this.simpleLabel;
 
-  toString() {
+  toString(group?: string) {
```

Comment:
This is too specific for `toString`

why not change `toVerboseString` accordingly?

Reply:
because other enums are using the toString for the status without the group..

Reply:
also toVerboseString was removed since it was not used

Reply:
if we only using the `group` in one place ?
we can do `{{status}} ({{group}})` there ?

Reply:
it is not the only place, so unless we do larger refactoring we need the group inside the status